Storage/api_types.h

```diff
@@ -34,10 +34,14 @@ namespace current {
 namespace storage {
 namespace rest {
 
+// Top-level URL prefixes for Storage REST.
 const std::string kRESTfulDataURLComponent = "data";
 const std::string kRESTfulSchemaURLComponent = "schema";
+const std::string kRESTfulCQRSCommandURLComponent = "command";
+const std::string kRESTfulCQRSQueryURLComponent = "query";
```

Comment:
Perhaps `/index/` for the generic query engine? You guys convinced me what we're doing is not really CQRS :)